package.json

```diff
@@ -279,6 +279,22 @@
           "description": "Set a custom folder path for cached XML Schemas. An absolute path is expected, although the ~ prefix (for the user home directory) is supported.",
           "scope": "window"
         },
+        "xml.server.binary.enabled": {
+          "type": "boolean",
+          "default": "false",
+          "description": "Use an experimental binary build of the LemMinX XML language server instead of the Java version. Additions to LemMinX provided by other extensions won't work.",
+          "scope": "window"
+        },
+        "xml.server.binary.path": {
+          "type": "string",
+          "description": "The path to the server binary to run. Will be ignored if `xml.server.binary.enabled` is not set. A binary will be downloaded if this is not set.",
+          "scope": "window"
```

Comment:
'scope":"machine"

Reply:
doing the same for the command line arguments to the binary